src/app/pages/asset-details/asset-details.component.html

```diff
@@ -223,34 +226,65 @@ <h5 class="heading-h3 bid-head" >{{auction.sellNowPrice}} ONE</h5>
-                                                <div class="input-main">
+                                            <form class="bidform" *ngIf="visible && !this.owner && currentTime < auctionTime && !sellPriceMet" [@slideInOut] #buyForm="ngForm" novalidate >
+<!--                                              {{ location.country_name }}-->
+                                               <div class="current-bid">
+                                                <div class="input-main" style="flex: 1">
                                                     <div *ngIf="auction.bids.length <= 0" class="input-otr-2">
                                                         <input class="input" style="width: 100%;
                                                         margin-bottom: 25px;
                                                         font-size: 20px;
-                                                        text-align: center; border-radius: 8px;
+                                                        text-align: left; border-radius: 8px;
                                                         background-color: white;" type="text" name="amount" [(ngModel)]="amount" placeholder="Current Bid: {{auction.highestBid}} ONE"  type="number" id="amount" name="amount"required>
                                                     </div>
                                                     <div *ngIf="auction.bids.length > 0" class="input-otr-2">
                                                         <input class="input" style="width: 100%;
                                                         margin-bottom: 25px;
                                                         background-color: white;
-                                                        text-align: center; border-radius: 8px"
+                                                        text-align: left; border-radius: 8px"
                                                                type="text" name="amount" [(ngModel)]="amount" placeholder="Current Bid: {{auction.bids[0]['bid']}} ONE ({{this.auctionValue | currency}}) "  type="number" id="amount" name="amount"required>
                                                     </div>
                                                 </div>
 
-                                                <div class="action-otr" *ngIf="visible && accountFound">
-                                                    <a (click)="bid()" class="btn-fill btn-bid">Place a Bid</a>
+                                                <div class="action-otr" style="flex: 1" *ngIf="visible && accountFound">
+                                                    <a (click)="bid()" class="btn-fill btn-bid" style="border-top-left-radius: 0; border-bottom-left-radius: 0;">Place a Bid</a>
+                                                </div>
+                                               </div>
```

Comment:
Please comment out stripe implementation as we will be pushing the develop build to live, once the setup is complete for stripe we can add it to the live build

Reply:
Requested updates implemented as the stripe codes have been commented out